@xen-orchestra/web/src/pages/vm/new.vue

```diff
@@ -190,11 +190,11 @@
                   </tr>
                 </thead>
                 <tbody>
-                  <tr v-for="(networkInterface, index) in vmState.networkInterfaces" :key="index">
+                  <tr v-for="(networkInterface, index) in visibleNetworkInterfaces" :key="index">
                     <td>
                       <!--        // Todo: Replace by the new select component -->
                       <div class="custom-select">
-                        <select v-model="networkInterface.interface">
+                        <select v-model="networkInterface.interface" @change="updateVifWithDevice(networkInterface)">
```

Comment:
```suggestion
  <select v-model="networkInterface.interface" >
```

you don't need to add `@change`